app/views/carts/show.html.erb

```diff
@@ -16,12 +16,18 @@
         <h6>Total: $<%= @cart.grand_total %></h6>
       </div>
     </div>
-        <% end %>
-      <% if @cart_contents != 0 %>
+    <% end %>
+    <% if @cart_contents != 0 %>
       <div class="empty-cart-contents">
         <%= button_to "Empty Cart Contents.", cart_path, method: :delete, class: 'empty-cart' %>
       </div>
-      <% end %>
-    </div>
+    <% end %>
+    <% if current_user %>
+      <%# <%= link_to '', checkout_path %> %>
```

Comment:
what is this line?

Reply:
or is it for later and just isn't fully commented out?

Reply:
it's just commented out for later